lib/ai/model/tool.go

```diff
@@ -26,6 +26,13 @@ import (
 
 	"github.com/gravitational/teleport/api/gen/proto/go/assist/v1"
 	"github.com/gravitational/teleport/api/types"
+	embeddinglib "github.com/gravitational/teleport/lib/ai/embedding"
+	"github.com/gravitational/teleport/lib/services"
+)
+
+const (
+	proxyLookupClusterMaxSize = 100
```

Comment:
Can you add a comment here? What is the max size?

Reply:
```suggestion
	// proxyLookupClusterMaxSize is max the number of nodes in the cluster to attempt an opportunistic node lookup
	// in the proxy cache. We always do embedding lookups if the cluster is larger than this number.
	proxyLookupClusterMaxSize = 100
```